### ISSUE TITLE
Usage of arguement " --options text_encoder_type=/path/to/bert-base-uncased"

### ISSUE THREAD

**Issue by Azure-107** [longzw1997/Open-GroundingDino#34]
Hi, I am wondering if you could elaborate a little bit more about the purpose and functionality of the argument `--options text_encoder_type=/path/to/bert-base-uncased`? Specifically under what circumstances one should we use it? I tried finetuning on an object detection dataset without visual grounding and noticed that even without including this argument, the training loss is still able to converge. 

Thank you so much!

**Comment by BIGBALLON** [comment]
@Azure-107 
1. the default ``text_encoder_type = "bert-base-uncased"``, which mean it will download [bert](https://huggingface.co/bert-base-uncased) model from huggingface auto. But if the cluster machine does not have a network, it can only read the existing local model. The purpose of this parameter is to specify the path of the model.
2. Bert if the cluster machine does not have a network, it can only read the existing local model. The purpose of this parameter is to specify the path of the model.

<img width="532" alt="image" src="https://github.com/longzw1997/Open-GroundingDino/assets/7837172/f51fb8d5-d0c6-47f6-ba4c-809da72eb665">

**Comment by Hasanmog** [comment]
Hello , 
I applied lora to bert  in the GroundingDINO and trained it on a custom dataset , but when evaluating I'm getting this warning. 
Any suggestions ? 
![image](https://github.com/longzw1997/Open-GroundingDino/assets/108143194/bf4dade9-991f-4926-abf2-b07081acfcd9)
